### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by janhavisinghh**

### CODE REVIEW

**Review by Honey14 — APPROVED** [review]
LGTM

**Review by Honey14 — CHANGES_REQUESTED** [review]
Some code climate issue, PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 - View next appointment information in patient summary screen
 
-## 2022-01-10-8093
+## 2022-01-10-8098
```
Comment (msasikanth) [inline CHANGELOG.md:16]: The version number is incorrect, can you update this, please? `2022-01-11-8098`

**Review by msasikanth — APPROVED** [review]

**Review by Honey14 — APPROVED** [review]

**Pull request merged**